README.md

```diff
@@ -6,6 +6,8 @@
 
 An MkDocs extension to generate documentation for Click command line applications.
 
+Originally developed by [DataDog](https://www.datadoghq.com/).
```

Comment:
Unfortunately I have been unable to get us to change our GitHub name because Go packages have a weird issue with case sensitivity and many depend on our stuff... but this is our actual branded spelling

```suggestion
Originally developed by [Datadog](https://www.datadoghq.com).
```